src/main/java/uk/gov/hmcts/reform/preapi/controllers/CaseController.java

```diff
@@ -126,4 +126,11 @@ public ResponseEntity<Void> undeleteCase(@PathVariable UUID id) {
         caseService.undelete(id);
         return ResponseEntity.ok().build();
     }
+
+    @PostMapping("/close-pending")
+    @Operation(operationId = "closePendingCases", summary = "Close cases in PENDING_CLOSURE state > 29 days")
+    public ResponseEntity<Void> closePending() {
+        caseService.closePendingCases();
+        return ResponseEntity.ok().build();
```

Comment:
Return a 204 instead of 200
```suggestion
        return ResponseEntity.noContent().build();
```